internal/credential/vault/repository_credentials.go

```diff
@@ -35,15 +35,16 @@ func (r *Repository) Issue(ctx context.Context, sessionId string, requests []cre
 
 	var creds []credential.Dynamic
 	var minLease time.Duration
+	var msgs []event.Option
+	runJobsInterval := r.scheduler.GetRunJobsInterval()
 	for _, lib := range libs {
 		cred, err := lib.retrieveCredential(ctx, op, sessionId)
 		if err != nil {
 			return nil, err
 		}
 
-		runJobsInterval := r.scheduler.GetRunJobsInterval()
 		if cred.getExpiration() < runJobsInterval {
-			event.WriteError(ctx, op, fmt.Errorf("WARNING: credential expires before job scheduler can run"), event.WithInfo("runJobsInterval", runJobsInterval))
+			msgs = append(msgs, event.WithInfo("credential_public_id", cred.GetPublicId()))
```

Comment:
We should add info about the credential library as well since that contains the information needed for the admin to update Vault to set an appropriate TTL